app-text/mupdf/mupdf-1.19.1.recipe

```diff
@@ -0,0 +1,134 @@
+SUMMARY="A lightweight XPS and PDF rendering library"
+DESCRIPTION="The renderer in MuPDF is tailored for high quality anti-aliased \
+graphics. It renders text with metrics and spacing accurate to within \
+fractions of a pixel for the highest fidelity in reproducing the look of a \
+printed page on screen.
+
+MuPDF is also small, fast, and yet complete. It supports PDF 1.7 with \
+transparency, encryption, hyperlinks, annotations, searching and more. It also \
+reads XPS/OpenXPS documents and CBZ (Comic Book archive) files."
+HOMEPAGE="https://www.mupdf.com/"
+COPYRIGHT="2006-2022 Artifex Software, Inc"
+LICENSE="AGPL-3.0"
+REVISION="1"
+SOURCE_URI="https://mupdf.com/downloads/archive/mupdf-$portVersion-source.tar.xz"
+CHECKSUM_SHA256="b5eac663fe74f33c430eda342f655cf41fa73d71610f0884768a856a82e3803e"
+SOURCE_DIR="mupdf-$portVersion-source"
+PATCHES="mupdf-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+# get the library version number (excluding patch rev) from port version
+libVersion=$(echo $portVersion | cut -d"." -f1,2)
+
+PROVIDES="
+	mupdf$secondaryArchSuffix = $portVersion
+	lib:libmupdf$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libjbig2dec$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libopenjp2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	mupdf${secondaryArchSuffix}_devel = $portVersion
+	devel:libmupdf$secondaryArchSuffix
```

Comment:
Same here:
```diff
- 	devel:libmupdf$secondaryArchSuffix
+ 	devel:libmupdf$secondaryArchSuffix = $libVersionCompat
```

Reply:
Looking at things this is not resolved? This needs to be done so pymupdf can pick up the correct version for mupdf.